### PR TITLE
More addiction snippets for nicotine and opiates.

### PR DESCRIPTION
**Pull request by Maleclypse**
#### Summary
None

#### Purpose of change
I saw too many repeat addiction snippets while crafting.  Decided to be the change.

#### Describe the solution
Adds more snippets.  Hopes to convince others to write more addiction snippets.

#### Describe alternatives you've considered
None

### CODE REVIEW

File: data/json/snippets/addiction.json [hunk 1]
```diff
@@ -112,7 +118,12 @@
       "Your hands start shaking… you need some painkillers.",
       "If you don't find some opiates soon you might start engaging in reckless behavior, you think to yourself.",
       "You daydream of crushing pills and snorting them through a rolled up dollar bill.  The drip in the back of your throat presaging a fun time.",
-      "If I can just find some oxies, everything will be fine."
+      "If I can just find some oxies, everything will be fine.",
+      "You can imagine it right now, that tingle starting at your extremities, warm and welcoming.  But it's not real, you haven't had any opiates today.  Why haven't you? The world already ended, you don't have to stay sober to talk to your family or a job.",
```
Comment (anothersimulacrum) [inline data/json/snippets/addiction.json:122]:
```suggestion
      "You can imagine it right now, that tingle starting at your extremities, warm and welcoming.  But it's not real, you haven't had any opiates today.  Why haven't you?  The world already ended, you don't have to stay sober to talk to your family or a job.",
```

**Comment by anoobindisguise** [comment]
I don't think the messages are supposed to be in first person. Also these don't really account for "oh I took too many painkillers for fighting zombies and now I'm addicted" but rather indicate a more lifelong thing. They should probably be more neutral.

**Comment by Maleclypse** [comment]
> I don't think the messages are supposed to be in first person. Also these don't really account for "oh I took too many painkillers for fighting zombies and now I'm addicted" but rather indicate a more lifelong thing. They should probably be more neutral. The existing ones aren't great either. Like the nicotine "sex with an old flame" dream for example is really prescriptive.

Feel free to make a conditional for messages requiring the hobby background if you'd like to.

File: data/json/snippets/addiction.json [hunk 2]
```diff
@@ -112,7 +118,12 @@
       "Your hands start shaking… you need some painkillers.",
       "If you don't find some opiates soon you might start engaging in reckless behavior, you think to yourself.",
       "You daydream of crushing pills and snorting them through a rolled up dollar bill.  The drip in the back of your throat presaging a fun time.",
-      "If I can just find some oxies, everything will be fine."
+      "If I can just find some oxies, everything will be fine.",
+      "You can imagine it right now, that tingle starting at your extremities, warm and welcoming.  But it's not real, you haven't had any opiates today.  Why haven't you?  The world already ended, you don't have to stay sober to talk to your family or a job.",
+      "Today is a good day for hydros.",
```
Comment (I-am-Erk) [inline data/json/snippets/addiction.json:123]:
```suggestion
      "Today is a good day for dillies.",
```

File: data/json/snippets/addiction.json [hunk 3]
```diff
@@ -140,7 +158,10 @@
     "text": [
       "You feel anxious.  You need your painkillers!",
       "The same old dream: a bottle of oxycodone, a zipper bag, a hammer, a flat surface and a dollar bill.",
-      "♪Got a bag of pills I've been dealin'♪"
+      "♪Got a bag of pills I've been dealin'♪",
+      "There is a puddle of sweat around you.",
+      "Counting and sorting my pills by strength, to each pill a pile and then on into a bottle.",
+      "Dream of being in a doctor's office.  The doctor doesn't trust the pain I feel and nothing else they prescribe works like oxy's."
```
Comment (I-am-Erk) [inline data/json/snippets/addiction.json:164]:
```suggestion
      "Dream of being in a doctor's office.  The doctor doesn't trust the pain I feel and nothing else they prescribe works like oxy."
```

File: data/json/snippets/addiction.json [hunk 4]
```diff
@@ -131,7 +143,13 @@
       "You feel anxious.  You need your painkillers!",
       "You need to find an unlooted pharmacy today before the lack of opiates kills you.",
       "The best thing about the apocalypse is that you've become your own prescriber.",
-      "The doctor is in and the doctor says it's time for some morphine, stat!"
+      "The doctor is in and the doctor says it's time for some morphine, stat!",
+      "So doctor, what should we take today to treat this existential post apocalyptic pain?",
+      "Will a veterinarian's office have opiates in it, I wonder?",
+      "Every day I don't take my pills is another day of the apocalypse wasted.",
+      "♪A little bit of hydro by my side, a little bit of oxy for a ride, a couple of trammy to even me out, and a little bit of morphine all night long.  Opiate Number 5!♪",
```
Comment (I-am-Erk) [inline data/json/snippets/addiction.json:150]:
```suggestion
      "♪A little bit of dilly by my side, a little bit of oxy for a ride, a couple of trammy to even me out, and a little bit of morphine all night long.  Opiate Number 5!♪",
```

**Review by I-am-Erk — COMMENTED** [review]
It may be regional, but "dilly" for dilaudid is a much more common slang for hydromorphone

File: data/json/snippets/addiction.json (hunk 2 above)
Comment (Maleclypse) [reply data/json/snippets/addiction.json:123]: Hydros is slang for acetaminophen and codeine. 
I didn’t think we had dilaudid in game?

Comment (I-am-Erk) [reply data/json/snippets/addiction.json:123]: I have never heard that one. Mostly there's no slang for that, it's used on the street but it's not that popular as a street drug. People call it t3 or tylenol

Comment (Maleclypse) [reply data/json/snippets/addiction.json:123]: I see that you are unfamiliar with the works of 3 6 Mafia.

Comment (Maleclypse) [reply data/json/snippets/addiction.json:123]: Definitely was in slang in the 90's to mid 2000's here's one of their many lyrical references to it.  https://www.youtube.com/watch?v=XTBWZw0iUp0

Might just be a USA thing.

Comment (Maleclypse) [reply data/json/snippets/addiction.json:123]: So you made me question was this just a southern thing, and I was like no surely Bone Thugs N Harmony also referred to them as hydros.  Yes, yes they did. https://www.youtube.com/watch?v=oXykr1BnveQ  Thank you for making me check my long held assumptions.

**Pull request closed without merging**